Sources/Framer/HTTPHandler.swift

```diff
@@ -67,12 +67,14 @@ public struct HTTPWSHeader {
         req.setValue(HTTPWSHeader.versionValue, forHTTPHeaderField: HTTPWSHeader.versionName)
         req.setValue(secKeyValue, forHTTPHeaderField: HTTPWSHeader.keyName)
         
-        if let cookies = HTTPCookieStorage.shared.cookies(for: url), !cookies.isEmpty {
-            let headers = HTTPCookie.requestHeaderFields(with: cookies)
-            for (key, val) in headers {
-                req.setValue(val, forHTTPHeaderField: key)
+		if req.allHTTPHeaderFields?["Cookie"] == nil {
+            if let cookies = HTTPCookieStorage.shared.cookies(for: url), !cookies.isEmpty {
```

Comment:
The indentation here looks odd, but maybe it's from the original repo commits?

Reply:
yes